apps/ui/src/composables/useSpaces.ts

```diff
@@ -143,6 +145,7 @@ export function useSpaces() {
   async function fetch(filter?: SpacesFilter) {
     if (loading.value) return;
     loading.value = true;
+    explorePageSpaces.value = [];
```

Comment:
This should be handled by `overwrite` flag that we pass as true.

We should instead update it here:
https://github.com/snapshot-labs/sx-monorepo/blob/9d5a60d970fba09e1c936a3c74a0f3a41f465580/apps/ui/src/composables/useSpaces.ts#L110

Without current change in this PR it's still possible to get duplicates if somehow we trigger another fetch before previous one finished.